Gaijinizer patch/Skills.txt

```diff
@@ -5145,7 +5145,7 @@ Untangle#afrg
 > BEGIN STRING
 \\N[1]の身体に瞬く間にネバネバの糸が巻き付いていく…！！
 > CONTEXT: Skills/411/message2
-\\N[1]'s body has become completly wraped in web!#afrgPR
+\\N[1]'s body has become completely wrapped in sticky web...!!#afrgPR/Strate
 > END STRING
```

Comment:
You'll need to check how many characters are displayed here during the fight. We don't want to make this line too long or else the end will get cut off.